.github/workflows/axosyslog-nightly.yml

```diff
@@ -9,7 +9,24 @@ on:
 
 
 jobs:
+  precheck:
+    runs-on: ubuntu-latest
+    outputs:
+      nightly-needed: ${{ github.event_name != 'schedule' || fromJson(steps.previous-nightly.outputs.data).workflow_runs[0].head_sha != github.sha }}
```

Comment:
I might be missing some GHA trickery here, but if I understand it correctly then for the firs run (when no previous hash is available) the failing property access (`.workflow_runs[0].head_sha`) will be converted into a `false` value?

Reply:
For future reference: https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-repository